### ISSUE TITLE
Deploy to AWS Lambda

### ISSUE THREAD

**Issue by gilesbutler** [smooch/smooch-bot-example#27]
Hi,

Would this bot work on a serverless architecture such as AWS Lambda?

Thanks

**Comment by alavers** [comment]
Though I haven't tried myself, I found this guide https://aws.amazon.com/blogs/compute/nodejs-packages-in-lambda/ which suggests that running a smooch-bot this way should be easy. You'd have to include the npm module dependencies as part of your Lambda deployment as the guide shows.

**Comment by gilesbutler** [comment]
Thanks for the response @alavers - I gave it a go after posting actually. I got it working using this new setup from AWS labs - [https://github.com/awslabs/aws-serverless-express](https://github.com/awslabs/aws-serverless-express) - it was actually fairly simple 👍 

The only problem is that the bot responses are broken, the first response is correct but then it seems to double up after or just get confused.

Any thoughts?

<img width="410" alt="screen shot 2016-12-19 at 12 30 32" src="https://cloud.githubusercontent.com/assets/790734/21298582/01a8609c-c5e7-11e6-86f3-d97e073f6495.png">
